src/metabase/api/search.clj

```diff
@@ -285,67 +286,67 @@
       (add-collection-join-and-where-clauses :card.collection_id search-ctx)
       (add-card-db-id-clause (:table-db-id search-ctx))))
 
-(s/defmethod search-query-for-model "action"
-  [model search-ctx :- SearchContext]
+(defmethod search-query-for-model "action"
```

Comment:
malli currently does not support typing for defmethod. But I don't think we lose anything here tho because this is a private multi and all function that calls this method has `search-ctx` schema already.